### ISSUE TITLE
Fix SNOMED high-level treatment domain assignments.

### ISSUE THREAD

**Issue by mgurley** [OHDSI/OncologyWG#49]

**Comment by dimshitc** [comment]
Actually, not only SNOMED but other Standard procedure terminologies like HCPCS, ICD9Proc, etc.
all these
Hormonotherapy, Chemotherapy, Antibioticotherapy, all these, will become Drugs.
So, I need to define the whole list of the concepts that need domain change and post it on the forum to check if it makes any problems for users.

**Comment by cgreich** [comment]
Сorrect

**Comment by mgurley** [comment]
Makes sense.  I changed the the task title to not be SNOMED-specific.

**Comment by sratwani** [comment]
Per Leadership call on 12/17, this task does not belong to any particular milestone and can be done at any point. Changing the milestone to 'creating standard vocabularies for diagnostic modifiers'

**Comment by dimshitc** [comment]
This looks like a big change of vocabulary and needs to be discussed in a broader community.
Any ETL will be affected.

**Comment by sratwani** [comment]
@cgreich @rimusia With this issue, the recommendation is to make change in ‘Drug’ domain definition. The idea behind this is that broad categories like chemotherapy, immunotherapy their concepts should have domain ‘drug’. We need approval from broader community that the ‘drug’ domain should have categories and not just purely drug information. Reaching out to you both to see if this is something you can bring up with the OMOP Modeling/CDM/Vocabulary team

**Comment by sratwani** [comment]
Copying @cgreich response. Here is the deal with this: The V6 is under stress. We are cleaning it up. Rapid changes are difficult at this point. The idea is that if changes are backwards compatible we would add them, and if they are not (changing something) than not. The Oncology Extension is of the former type (just adding tables and fields). Let me talk to Clair.

**Comment by cgreich** [comment]
So, here is the situation:
1. Currently, a drug is anything with a specified Ingredient, or more detail. Everything that puts Ingredients in a bucket is a Drug Class, which cannot be used in the data.
2. This is arbitrary. We could extend the Drug Domain up.
3. If we do that, we need to pick a vocabulary that is now a classification system and will become a Standard. ATC is the only contender, because it is international and has the biggest traction.
4. We are finalizing (after years of futzing around) the ATC to RxNorm hierarchy rework. That was needed because ATC is "dirty". It incorporates attributes we have difficulties with (e.g. "indication of the drug")
5. Taking an arbitrary "Chemotherapy" or "Chemo or Immunotherapy" and promoting it just like that is not a good idea. Because there is no connection to actual drugs that constitute being a chemotherapeutic agent.
6. We could use HemOnc, but it is also maturing, and it is only cancer.

My bottom line: Let's finish ATC, look at the chemo and immuno classes, check them out, and push the proposal.

It actually has nothing to do with V6 or Clair. My mistake of confusing the issue.

**Comment by jeremywarner** [comment]
Just a remark about using HemOnc, as you say it is maturing and it is only cancer. Also important to note that the “high-level” classes (chemotherapy, immunotherapy, etc), which we actually capture separately as modalities, are anchored to the regimen not to the drug. This is because some drugs have different actions in different contexts. For example:

Prednisone the drug is a steroid is a drug

Prednisone the component of R-CHOP is a chemotherapy
Prednisone the component of Docetaxel & Prednisone is a hormonotherapy

Best,
Jeremy

Jeremy L. Warner MD, MS, FAMIA
Associate Professor of Medicine and Biomedical Informatics
Associate Editor, JCO Clinical Cancer Informatics
Deputy Editor, HemOnc.org
2220 Pierce Avenue | 777 PRB | Nashville, TN  37232-6307

Jennifer Bauer
Interim Administrative Assistant
jennifer.m.bauer@vumc.org
615-875-4547

**Comment by cgreich** [comment]
Exactly. ATC does that too, btw. In fact, ATC has it as an antiinflammatory ([A07EA03](https://www.whocc.no/atc_ddd_index/?code=A07EA03)) and an endocrine ([H02AB07](https://www.whocc.no/atc_ddd_index/?code=H02AB07)) drug. It doesn't have it as an antineoplastic, though. That's the crux. These classes are not precise and commonly agreed upon, which is why I have resisted for all that time to include them as concepts for data. But if we bless one to be the standard we will have to live with the choices made in that classification system and stop whining. Those are the choices.

**Comment by sratwani** [comment]
Michael and Rimma are writing a proposal with explicit references for the group to review. Below are the areas being covered in the proposal
a.	High level treatment concepts (Chemo, Radiation, Surgery, Immunotherapy, Targeted Therapy etc.) in order to produce episodes about the treatment. 
b.	New domain for the Treatment Regimens
c.	Once the high-level concepts are added to the vocabulary ->
For concepts from Cancer Registry, it will directly map to top level concepts (defined above). 
Where there is no registry information and only Claims/EMR information is available, there is a task to link drugs to the high-level concepts that we are creating, put them in hierarchy with the highest level as being identified above.

**Comment by sratwani** [comment]
@mgurley For this issue 'a' and 'b' below are complete. 'c' low level concepts need to be mapped. You mentioned in the Dev call that Rimma's team is working on this,
a. High level treatment concepts (Chemo, Radiation, Surgery, Immunotherapy, Targeted Therapy etc.) in order to produce episodes about the treatment.
b. New domain for the Treatment Regimens
c. For concepts from Cancer Registry, it will directly map to top level concepts (defined above).
Where there is no registry information and only Claims/EMR information is available, there is a task to link drugs to the high-level concepts that we are creating, put them in hierarchy with the highest level as being identified above.
